src/internal/handle_file_operation_test.go

```
@@ -165,3 +166,233 @@ func TestCompressSelectedFiles(t *testing.T) {
+
+			// Verify results based on whether paste should be prevented
+			if tt.shouldPreventPaste {
+				verifyPreventedPasteResults(t, &p.m, originalPath)
+				require.False(t, tt.shouldClipboardClear)
+			} else {
+				verifySuccessfulPasteResults(t, tt.targetDir, tt.expectedDestFiles, originalPath, tt.shouldOriginalExist)
+			}
+			assert.False(t, ListeningMessage)
+			slog.Debug("model clipboard info 4", "items", p.m.copyItems.items, "cut", p.m.copyItems.cut)
+
+			time.Sleep(2*time.Second)
```

golangci-lint (1.64.8)

File is not properly formatted

(goimports)